dynawo/sources/Common/CMakeLists.txt

```diff
@@ -111,9 +111,10 @@ add_library(dynawo_Common SHARED ${COMMON_SOURCES})
 add_dependencies(dynawo_Common create_keys_files)
 
 target_include_directories(dynawo_Common
+  PUBLIC
```

Comment:
How come we need to export it when it is used internally to Commoin library now?

Reply:
If we use the INTERFACE scope like before, we can only include config.h and gitversion.h in header only files. I just needed to include config.h and gitversion.h in a .cpp file which corresponds to the PRIVATE scope. So I put the PUBLIC scope to use both INTERFACE and PRIVATE scope at once.

Reply:
PRIVATE should be enough no?

Reply:
It's not enough, because other targets need to include DYNInitXml.h which includes config.h

Reply:
ok thanks